applications/vanilla/controllers/class.vanillasettingscontroller.php

```diff
@@ -40,11 +40,25 @@ public function posting() {
         // Check permission
         $this->permission('Garden.Settings.Manage');
 
+        $formats = ['Text']; // Check for additional formats
+        $this->EventArguments['formats'] = &$formats;
+
+        $this->fireEvent('GetFormats');
```

Comment:
Since we're only modifying a single value, what do you think about using `fireFilter` instead?